content/themes/custom/models/search.php

```diff
@@ -402,18 +397,23 @@ protected function get_post_data( stdClass $result, stdClass $params ) {
      *
      * @param  string $guid Item guid.
      * @param  array  $tree Api item tree.
-     * @param  string $key  Term key to search for.
      * @return mixed        Term or null on no match.
      */
-    protected function get_post_term( string $guid, array $tree, string $key = 'taskgroup_term' ) {
-        $item = $tree[ $guid ] ?? [];
-
-        if ( ! empty( $item[ $key ] ) ) {
-            return $item[ $key ];
+    protected function get_post_term( string $guid, array $tree ) {
```

Comment:
This works well. Another way of getting the term for an activity would be to store it in the original item array as under a consistent key. It could be accessed as follows: `$item['term']`.